Sources/BraintreeCoreSwift/BTAPIClient.swift

```diff
@@ -22,6 +26,14 @@ import Foundation
     var apiHTTP: BTAPIHTTP?
     var graphQLHTTP: BTGraphQLHTTP?
 
+    /// Exposed for testing analytics
+    /// By default, the `BTAnalyticsService` instance is static/shared so that only one queue of events exists.
+    /// The "singleton" is managed here because the analytics service depends on `BTAPIClient`.
+    var analyticsService: BTAnalyticsService? {
+        get { BTAPIClient._analyticsService }
+        set { BTAPIClient._analyticsService = newValue }
+    }
+
```

Comment:
_Take it or leave it comment_ - moving the private analtyicsService property to be right below the internal one for testing.

```suggestion
    }

    private static var _analyticsService: BTAnalyticsService?
```